src/LiveSplit.Core/Model/RunMetadata.cs

```diff
@@ -117,6 +117,49 @@ public IDictionary<Variable, VariableValue> VariableValues
             }
         }
 
+        public class CustomVariable
+        {
+            public string Value = null;
```

Comment:
Do not expose fields publicly.

```suggestion
            public string Value { get; set; }
```